pkg/cargo/kilnfile.go

```diff
@@ -36,8 +36,10 @@ func (kf *Kilnfile) BOSHReleaseTarballSpecification(name string) (BOSHReleaseTar
 
 func (kf *Kilnfile) Glaze(kl KilnfileLock) error {
 	kf.Stemcell.Version = kl.Stemcell.Version
+	var SkipGlazeReleases []string
 	for index, spec := range kf.Releases {
 		if spec.FloatAlways {
+			SkipGlazeReleases = append(SkipGlazeReleases, spec.Name)
```

Comment:
week opinion: consider logging the skip and maybe setting a boolean "hasAlwaysFloatReleases" instead of creating another list... I don't see any other use of the list.

Reply:
@crhntr Can you please elaborate? I am super new to Go

Reply:
Sure :) You seem to be adding elements to a list to them immediately log the lines out. Doing so adds an extra data structure (a simple slice) that you may not need. You could get the same result with logging in the `if spec.FloatAlways {}` block. If you really want to add the "all releases were glazed" log line, you could keep that functionality by assigning "skippedSomeValue = true" or something like that then conditionally logging, although I don't think that tells me anything useful that I don't already know from the other log lines.

Reply:
Ha yes. That makes total sense. When I am working on kiln code, I tend to get lost and don't think straight lol. Thank you.

Reply:
Ok at first I was wondering why I didn't think of this. But now I know. 
I want to display the message "Skipping glaze for the following releases:" only if it skips pinning releases.
And I won't know that until it processes all the releases in the Kilnfile. That's why I am not printing the skipped releases in the spec.FloatAlways {} block.